nextflow_schema.json

```diff
@@ -1,1697 +1,1816 @@
+                },
+                "preseq_maxextrap": {
+                    "type": "integer",
+                    "default": 10000000000,
+                    "description": "Specify the maximum extrapolation (lc_extrap mode only)",
+                    "fa_icon": "fas fa-ban",
+                    "help_text": "Specify the maximum extrapolation that `lc_extrap` mode will perform.\n\n> Modifies preseq lc_extrap parameter: `-e`"
+                },
+                "preseq_terms": {
+                    "type": "integer",
+                    "default": 100,
+                    "description": "Specify the maximum number of terms for extrapolation (lc_extrap mode only)",
+                    "fa_icon": "fas fa-sort-numeric-up-alt",
+                    "help_text": "Specify the maximum number of terms that `lc_extrap` mode will use.\n\n> Modifies preseq lc_extrap parameter: `-x`"
+                },
+                "preseq_bootstrap": {
+                    "type": "integer",
+                    "default": 100,
+                    "description": "Specify number of bootstraps to perform (lc_extrap mode only)",
+                    "fa_icon": "fab fa-bootstrap",
+                    "help_text": "Specify the number of bootstraps `lc_extrap` mode will perform to calculate confidence intervals.\n\n> Modifies preseq lc_extrap parameter: `-n`"
+                },
+                "preseq_cval": {
+                    "type": "number",
+                    "default": 0.95,
+                    "description": "Specify confidence interval level (lc_extrap mode only)",
+                    "fa_icon": "fas fa-check-circle",
+                    "help_text": "Specify the allowed level of confidence intervals used for `lc_extrap` mode.\n\n> Modifies preseq lc_extrap parameter: `-c`"
+                }
+            },
+            "fa_icon": "fas fa-bezier-curve",
+            "help_text": "nf-core/eager uses Preseq on mapped reads as one method to calculate library\ncomplexity. If DeDup is used, Preseq uses the histogram output of DeDup,\notherwise the sorted non-duplicated BAM file is supplied. Furthermore, if\npaired-end read collapsing is not performed, the `-P` flag is used."
+        },
+        "adna_damage_analysis": {
+            "title": "(aDNA) Damage Analysis",
+            "type": "object",
+            "description": "Options for calculating and filtering for characteristic ancient DNA damage patterns.",
+            "default": "",
+            "properties": {
+                "damage_calculation_tool": {
+                    "type": "string",
+                    "default": "damageprofiler",
+                    "description": "Specify the tool to use for damage calculation.",
+                    "fa_icon": "fas fa-tools",
+                    "help_text": "Specify the tool to be used for damage calculation. Options: `damageprofiler`, `mapdamage`. By default, DamageProfiler is used.",
```

Comment:
Explain when use either, and remove 'options:'
 because tha'ts in the enum

Reply:
This is DSL1, we have `Options:` and `Default:` in all other options...